tests/kola/version/rhel-major-version

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+# kola: { "exclusive": false }
+set -xeuo pipefail
+
+fatal() {
+    echo "$@" >&2
+    exit 1
+}
+
+# checks to ensure that the only packages from the RHEL major version are included
+
+source /etc/os-release
+
+
+RHEL_VERSION="8"   
+
+for x in $(rpm -qa --queryformat='%{RELEASE}\n' | grep -oP 'el\s*\K\d+')
+    do
+        if "${RHEL_VERSION} != "${x}""; then
```

Comment:
This branch is failing when run locally.  

I did `cosa run` to get an RHCOS 4.11 system up, downloaded this script, and ran it manually:

```
[core@cosa-devsh ~]$ rpm-ostree status
State: idle
Deployments:
* b97794784366ba8e34978aada1a32007dcd5cc4256b507696f2857c3f6e88a8c
                   Version: 411.86.202204131300-0 (2022-04-13T13:03:08Z)
[core@cosa-devsh ~]$ curl -LO https://raw.githubusercontent.com/openshift/os/53ac06f6402cf2cf494a04078f7988b087c51997/tests/kola/version/rhel-major-version
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   450  100   450    0     0   7377      0 --:--:-- --:--:-- --:--:--  7377
[core@cosa-devsh ~]$ chmod +x rhel-major-version 
[core@cosa-devsh ~]$ ./rhel-major-version                                                                                                                                                                                                                                                                                     
+ source /etc/os-release                                                  
++ NAME='Red Hat Enterprise Linux CoreOS'                                 
++ ID=rhcos                                                               
++ ID_LIKE='rhel fedora'                                                  
++ VERSION=411.86.202204131300-0                                          
++ VERSION_ID=4.11                                                        
++ PLATFORM_ID=platform:el8                                               
++ PRETTY_NAME='Red Hat Enterprise Linux CoreOS 411.86.202204131300-0 (Ootpa)'
++ ANSI_COLOR='0;31'                                                      
++ CPE_NAME=cpe:/o:redhat:enterprise_linux:8::coreos
++ HOME_URL=https://www.redhat.com/
++ DOCUMENTATION_URL=https://docs.openshift.com/container-platform/4.11/
++ BUG_REPORT_URL=https://bugzilla.redhat.com/
++ REDHAT_BUGZILLA_PRODUCT='OpenShift Container Platform'
++ REDHAT_BUGZILLA_PRODUCT_VERSION=4.11
++ REDHAT_SUPPORT_PRODUCT='OpenShift Container Platform'
++ REDHAT_SUPPORT_PRODUCT_VERSION=4.11
++ OPENSHIFT_VERSION=4.11
++ RHEL_VERSION=8.6
++ OSTREE_VERSION=411.86.202204131300-0
+ RHEL_VERSION=8
++ rpm -qa '--queryformat=%{RELEASE}\n'
++ grep -oP 'el\s*\K\d+'
+ for x in $(rpm -qa --queryformat='%{RELEASE}\n' | grep -oP 'el\s*\K\d+')
+ '8 != 8'
./rhel-major-version: line 19: 8 != 8: command not found
+ for x in $(rpm -qa --queryformat='%{RELEASE}\n' | grep -oP 'el\s*\K\d+')
+ '8 != 8'
...
[core@cosa-devsh ~]$ echo $?
0
```

Since the `if` statement fails, it never drops into the code where failure would be reported and the whole script exits successfully.

This is giving you a false positive result.